spec/spec_helper.rb

```diff
@@ -0,0 +1,23 @@
+ENV['HATCHET_BUILDPACK_BASE'] = 'https://github.com/heroku/heroku-buildpack-python.git'
```

Comment:
@schneems Does the hatchet rake task pick up the git branch?

Reply:
Git branch is auto detected on travis and heroku CI. If it's not detected on another CI provider let me know and I can add it. https://github.com/heroku/hatchet/blob/master/lib/hatchet/app.rb#L9 and https://github.com/heroku/hatchet/blob/master/lib/hatchet.rb#L28